### PR TITLE
Fixes guild skill delay call on map change

### LINKED ISSUE #5278: guild_block_skill
* **rAthena Hash**: 

* **Client Date**:

 Not aplicable

* **Server Mode**: 

Not aplicable

* **Description of Issue**: 
  * Result:  The function guild_block_skill is called when refresh client
  * Expected Result:  guild_block_skill called only once time, at login
  * How to Reproduce:  change map or refresh client
  * Official Information: Not aplicable
  
Commit 4944 include the function "clif_loadConfirm", that call 'guild_member_joined', and 'guild_member_joined' call 'guild_block_skill'
* **Modifications that may affect results**: 
  
  
Not aplicable?

### PR DESCRIPTION
**Pull request by aleos89**
* **Addressed Issue(s)**: Fixes #5278

* **Server Mode**: Pre-renewal and Renewal

* **Description of Pull Request**: 
  * Fixes guild masters triggering the guild skill delay call on each map change.
Thanks to @mplaten!

### CODE REVIEW

File: src/map/guild.cpp [hunk 1]
```diff
@@ -802,7 +802,7 @@ void guild_member_joined(struct map_session_data *sd) {
 		guild_request_info(sd->status.guild_id);
 		return;
 	}
-	if (strcmp(sd->status.name,g->master) == 0) {	// set the Guild Master flag
+	if (sd->state.connect_new == 1 && strcmp(sd->status.name,g->master) == 0) {	// set the Guild Master flag
```
Comment (Lemongrass3110) [inline src/map/guild.cpp:805]: This is probably related to the problem with the clan counter, because this function is called too often now.

Comment (Lemongrass3110) [reply src/map/guild.cpp:805]: See #5204

**Review by lighta — APPROVED** [review]

File: src/map/guild.cpp (hunk 1 above)
Comment (Triedge) [reply src/map/guild.cpp:805]: Uhm... this bug return!!!

Comment (Triedge) [reply src/map/guild.cpp:805]: Or more specifically, it seems that this fix was not applied. At least the change history in guild.cpp says it was never implemented.

**Comment by Lemongrass3110** [comment]
Closing this in favor of #5204

**Pull request closed without merging**